tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -343,6 +343,210 @@ def _calc_q(n, m, sign, k=1):
       (n - 1) ** 2 + tf.math.divide_no_nan(4 * m, k))) / 2
 
 
+def bjerksund_stensland_1993(volatilities,
+                 strikes,
+                 expiries,
+                 spots=None,
+                 forwards=None,
+                 discount_rates=None,
+                 dividend_rates=None,
+                 cost_of_carries=None,
+                 discount_factors=None,
+                 is_call_options=None,
+                 dtype=None,
+                 name=None):
+  """Computes the Bjerksund-Stensland 1993 price for a batch of call or put options.
+
+  #### Example
```

Comment:
please add a sentence saying that this is from 1993 paper. In the next CL you could expand that 2002 implementation is also supported

Reply:
Sure.